README.md

```diff
@@ -319,6 +319,8 @@ The settings exposed here are targeted to more advanced users that want to custo
 |MaxPartitionFetchBytes|max.partition.fetch.bytes|Trigger
 |FetchMaxBytes|fetch.max.bytes|Trigger
 |AutoCommitIntervalMs|auto.commit.interval.ms|Trigger
+|LibkafkaDebug|debug|Both
+|TopicMetadataRefreshFastIntervalMs|topic.metadata.refresh.interval.ms|Both

```

Comment:
According to the docs [here](https://kafka.apache.org/08/documentation.html) this seems to be applied to producers only. Does it affect the trigger as well?

Reply:
Dear Francisco,

configuring the metadata refresh interval appropriately is needed to mitigate nasty issues with Azure LBs, see also https://github.com/edenhill/librdkafka/issues/3109. This is especially important for consumers breaking down after four minutes of idle connection time. This has made many people exceptionally sad in the past.

I don't know how triggers will be handled internally, but I assume there will be some consumer around pushing the updates to the trigger?

With kind regards,
Andreas.

Reply:
Apologies if I got something wrong here. I was referring to the librdkafka configuration setting `metadata.max.age.ms`, which is applicable to both producers and consumers, see [1].

[1] https://docs.microsoft.com/en-us/azure/event-hubs/apache-kafka-configurations#librdkafka-configuration-properties

Reply:
As amotl said, it looks both. However, I remove this property since it is not needed.